build.gradle

```diff
@@ -242,6 +342,7 @@ project('serializers') {
 
     jar {
         manifest {}
+        zip64=true
```

Comment:
Please add a comment about why this was needed.

Reply:
number of zipped files was exceeding 65k and gradle required this flag to be set to true.